js/script.js

```diff
@@ -583,3 +535,159 @@ document.addEventListener('DOMContentLoaded', async (event) => {
         await checkConnected();
     }
 });
+
```

Comment:
What with all this white-space?

Reply:
I think that's when I started to separate layout-related JS code from the general workflow business logic.
In a follow-up commit 6f31942cb34f17ae3432e9c6915c80d65562197d I moved that entire bottom section into a new file to complete this separation.